docs/view/StarRating/README.md

```diff
@@ -22,13 +23,24 @@ Example of a Star Rating in Kotlin
 
 ```Kotlin
 import net.skyscanner.backpack.starrating.BpkStarRating
+import net.skyscanner.backpack.starrating.RoundingType
 
 BpkStarRating(context).apply {
   maxRating = 5
   rating = 2.5f
+  rounding = RoundingType.UP
 }
 ```
 
+# Props
```

Comment:
I saw this on the web docs and thought it'd be very helpful for this component particularly to know the properties & default values